operations/mimir-mixin-tools/serve/run.sh

```diff
@@ -6,7 +6,7 @@ set -e
 SCRIPT_DIR=$(cd `dirname $0` && pwd)
 # Ensure we run recent Grafana.
 GRAFANA_VERSION=11.0.0
-DOCKER_CONTAINER_NAME="mixin-serve-grafana-graph"
+DOCKER_CONTAINER_NAME="mixin-serve-grafana"
```

Comment:
i think this was a typo in a previous PR. this line expects the old URL https://github.com/grafana/mimir/blob/cf1d7a81033b8024c7b916bd26f913cb5f9ff00d/operations/mimir-mixin-tools/screenshots/app.js#L79